### ISSUE TITLE
Desktop Electron client need some attention

### ISSUE THREAD

**Issue by LFCavalcanti** [RocketChat/Rocket.Chat#4083]
Sorry to create an issue to address this, but we already asked on Chat, on commit discussions before and feel like the desktop client is abandoned by you guys.

There are problems with the download link, the version for Windows i386 and 4 PR sitting there waiting approval since April 29! @engelgabriel  did commit to merge scripts to generate an RPM package, but that's about it.

I don't want to sound like bitching, but we don't have access to merge, commit and accept those PR, so PLEASE, can we get some attention over there?

**Comment by geekgonecrazy** [comment]
@LFCavalcanti definitely not abandoned.  We apologize for the delay on things.  We've been working on some pretty big stuff here in the main project.

Can you point me to some specific things you'd like us to take a look at?

**Comment by LFCavalcanti** [comment]
Hi Aaron,

As I said, there are 4 PRs ready there. Plus, I remember we "chatted" back in June about the Electron client not working correctly on Windows 7 32Bits, you said there was already a fix for it and just needed the time to commit it. So here I'm still waiting...

Again, you guys are doing a awesome job here, it's just that since you hold the keys to the repo, we can only wait.

I get frustrated when I have to talk like this, because I'm not paying you to code and feels I'm bitching more than trying to help, but in the other hand, if someone from the "core team" doesn't process and merge those PR and release the updated binaries, they will stay there waiting.

Thanks for taking the time to read and sorry if I sound like bitching or whining.

**Comment by geekgonecrazy** [comment]
@LFCavalcanti I understand the frustration of feeling like something is not moving.  I really do.  I've been the outside guy waiting on the maintainer to merge a badly needed pull request.

But we can't always merge every pull request.  Especially if we think it will introduce problems.

https://github.com/RocketChat/Rocket.Chat.Electron/pull/199#issuecomment-241465426 See my response on this one for this particular example.

If you have others let me know.

**Comment by LFCavalcanti** [comment]
I already saw your reply, I'll download his repo and try to add a check to identify the system and only use the identification on Win7 and Vista.

**Comment by geekgonecrazy** [comment]
I'll go ahead and close this issue.  Thanks for taking a stab.  👍  Just mention a couple of us in your new pull request, and we'll give it a test and get it merged.
